WordPress/src/main/java/org/wordpress/android/ui/reader/repository/ReaderRepositoryEvents.kt

```diff
@@ -0,0 +1,31 @@
+package org.wordpress.android.ui.reader.repository
```

Comment:
Have you considered using data classes? For example `PostLikeSuccess, PostLikeFailure ...` could be data classes, right?

Reply:
Ohh I thought I did make them data classes. In commit 46d91a6

Reply:
I take that back, I originally had them as data classes, then switched them to class so I can use the `PostLikeEnded` event in the callback without having to check for each individual class type. I'll noodle on this to see if I can handle a different way. I like the use of class name instead of a value in the payload. We'll see, but yes, it was intentional not to use a data class

Reply:
>so I can use the PostLikeEnded event in the callback without having to check for each individual class type

I'm not sure I understand this statement. However, as you said - "We'll see".